app/questionnaire/placeholder_transforms.py

```diff
@@ -315,3 +315,9 @@ def option_label_from_value(self, value: str, answer_id: str) -> str:
         label may be simple string or a resolved placeholder string
         """
         return self.ops_helper.get_option_label_from_value(value, answer_id)
+
+    @staticmethod
+    def conditional_trad_as(trad_as: str) -> str:
+        if trad_as:
+            return f" ({trad_as})"
+        return ""
```

Comment:
could be one liner if you like it 
```suggestion
       return  f" ({trad_as})" if trad_as else ""        
```